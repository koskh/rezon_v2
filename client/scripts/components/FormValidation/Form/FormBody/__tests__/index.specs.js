import * as React from 'react';

import renderer from 'react-test-renderer';

import FormBody from '../';

describe('components/Form/FormBody', () => {

    test('Component can render', () => {
        const component = renderer.create(<FormBody />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

});
