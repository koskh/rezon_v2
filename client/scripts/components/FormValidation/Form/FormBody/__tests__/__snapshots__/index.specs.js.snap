// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Form/FormHeader Component can render 1`] = `
<div
  className="FormHeader"
>
  TestTitle Form Header
</div>
`;
